kura/emulator/org.eclipse.kura.emulator.position/META-INF/MANIFEST.MF

```diff
@@ -11,7 +11,7 @@ Bundle-ActivationPolicy: lazy
 Import-Package: javax.xml.parsers,
  org.eclipse.kura;version="[1.0,2.0)",
  org.eclipse.kura.configuration;version="[1.1,2.0)",
- org.eclipse.kura.position;version="[1.1,1.2)",
+ org.eclipse.kura.position;version="[1.0,2.0)",
```

Comment:
Emulator.position is providing an implementation for the position APIS. It should be a strict range in the form [1.2,1.3)